GuysNight.LethalCompanyMod.BalancedItems/ItemOverridesContainer.cs

```diff
@@ -5,71 +5,71 @@ namespace GuysNight.LethalCompanyMod.BalancedItems {
 		public static Dictionary<string, OverrideProperties> ItemOverrides { get; } = new Dictionary<string, OverrideProperties> {
 			//spawned scrap items
-			{ "7Ball", new OverrideProperties(0.5f, 135) },
-			{ "Airhorn", new OverrideProperties(1.5f, 155) },
-			{ "Bell", new OverrideProperties(5f, 160) },
-			{ "BigBolt", new OverrideProperties(7f, 65) },
-			{ "BottleBin", new OverrideProperties(125) },
-			{ "Brush", new OverrideProperties(1f, 55) },
-			{ "Candy", new OverrideProperties(1f, 53) },
-			{ "CashRegister", new OverrideProperties(20f, 300) },
-			{ "ChemicalJug", new OverrideProperties(20f, 145) },
-			{ "ClownHorn", new OverrideProperties(1f, 155) },
-			{ "Cog1", new OverrideProperties(25f, 115) },
-			{ "ComedyMask", new OverrideProperties(2f, 100) },
-			{ "Dentures", new OverrideProperties(1.5f, 180) },
-			{ "DiyFlashbang", new OverrideProperties(1.5f, 48) },
-			{ "DustPan", new OverrideProperties(1f, 55) },
-			{ "EggBeater", new OverrideProperties(0.5f, 70) },
-			{ "EnginePart1", new OverrideProperties(30f, 95) },
-			{ "FancyCup", new OverrideProperties(4f, 150) },
-			{ "FancyLamp", new OverrideProperties(10f, 235) },
-			{ "FancyPainting", new OverrideProperties(230) },
-			{ "FishTestProp", new OverrideProperties(0.5f, 85) },
-			{ "FlashLaserPointer", new OverrideProperties(1f, 165) },
-			{ "Flask", new OverrideProperties(2f, 75) },
-			{ "GiftBox", new OverrideProperties(50) },
-			{ "GoldBar", new OverrideProperties(27.5f, 390) },
-			{ "Hairdryer", new OverrideProperties(200) },
-			{ "LungApparatus", new OverrideProperties(50f, 80) },
-			{ "MagnifyingGlass", new OverrideProperties(4f, 130) },
-			{ "MetalSheet", new OverrideProperties(7f, 40) },
-			{ "MoldPan", new OverrideProperties(3f, 65) },
-			{ "Mug", new OverrideProperties(2f, 115) },
-			{ "PerfumeBottle", new OverrideProperties(0.5f, 190) },
-			{ "Phone", new OverrideProperties(140) },
-			{ "PickleJar", new OverrideProperties(3f, 115) },
-			{ "PillBottle", new OverrideProperties(1f, 70) },
-			{ "Remote", new OverrideProperties(1f, 85) },
-			{ "Ring", new OverrideProperties(0.2f, 165) },
-			{ "RobotToy", new OverrideProperties(5f, 180) },
-			{ "RubberDuck", new OverrideProperties(0.5f, 128) },
-			{ "SodaCanRed", new OverrideProperties(0.5f, 135) },
-			{ "SteeringWheel", new OverrideProperties(5f, 60) },
-			{ "StopSign", new OverrideProperties(6f, 90) },
-			{ "TeaKettle", new OverrideProperties(5f, 110) },
-			{ "Toothpaste", new OverrideProperties(1f, 78) },
-			{ "ToyCube", new OverrideProperties(1f, 85) },
-			{ "TragedyMask", new OverrideProperties(2f, 100) },
-			{ "WhoopieCushion", new OverrideProperties(0.5f, 33) },
-			{ "YieldSign", new OverrideProperties(6f, 90) },
+			{ "7Ball", new OverrideProperties(0.5f) },
+			{ "Airhorn", new OverrideProperties(1.5f) },
+			{ "Bell", new OverrideProperties(5f) },
+			{ "BigBolt", new OverrideProperties(7f) },
+			// { "BottleBin", new OverrideProperties(18f) },
```

Comment:
I want to spend some time thinking on how I want to handle the commenting out of these items. They are commented out because there is no override specified. But I want to keep them here for tracking purposes of what the item name is. But is there really any value in that? They're easily discoverable, just with a bit of manual effort. Should I have them uncommented, but use the default ctor for `OverrideProperties`? Probably.